src/utils/MessageMixin.js

```diff
@@ -113,7 +113,7 @@ export default {
                  * Runs every one second to set the duration passed before
                  * the alert will auto close to show it in the progress bar (Remaining Time)
                  */
-                this.progress = setInterval(() => {
+                window.durationProgress = setInterval(() => {
```

Comment:
Window??

Reply:
to use it outside of the vue instance of `MessageMixin.vue` in the `NoticeMixin.vue`.
made it globally

Reply:
Vue instance? window is really terrible :(

Reply:
@jtommy Why?
Do you have any alternative way to prevent it?

Reply:
As i told you might to attach it to a global variable of vue instance

Reply:
I'll try it.

Reply:
@jtommy I'll implement it today.

Reply:
@jtommy if I update the global var in `MessageMixin.vue` with the value of `setInterval`, it doesn't update when I use it in `NoticeMixin.vue`, why that happens?

Reply:
Mixins is not a singleton so it's the same instance

Reply:
I Have created a file in utils folder called **GlobalStore** (_acts like vuex_) to put any global variable to use through the vue components, so I put my variable to it and imported this **var** in `NoticeMixin` and `MessageMixin` and it that worked, the `vue.prototype` doesnt not worked as expected for me. might be used to put other variables in future by some devs

Reply:
@jtommy I Have created a file in utils folder called **GlobalStore** (_acts like vuex_) to put any global variable to use through the vue components, so I put my variable to it and imported this **var** in `NoticeMixin` and `MessageMixin` and it that worked, the `vue.prototype` doesnt not worked as expected for me. might be used to put other variables in future by some devs

Reply:
Why didn't work prototype, you can use global $buefy variable

Reply:
If I set the for instance `Vue.prototype.$example = null` in` index.js`, in MessageMixin.vue I assign the ID value of interval successfully, but when I want to use that variable in `NoticeMixin.vue` I found that it's value is `null`